index.js

```diff
@@ -125,7 +125,7 @@ request.defaults = function (options, requester) {
 }
 
 request.forever = function (agentOptions, optionsArg) {
-  var options = {}
+  const options = {}
```

Comment:
What do all these var to const replacements have to do with upgrading dependencies?

Did you run a linter or something?